### PR TITLE
remove faker which is 86M and not really used at all in live code

### PR DESCRIPTION
**Pull request by schuyler1d**
# Fixes build failing on deploy to AWS Lambda

## Description

The instructions in DEPLOYING_AWS_LAMBDA.txt don't work with current dependencies which are too large.  I remove `faker` which was used only in old files to generate a phone number easily 'faked' manually.

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [na] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [na] I have made any necessary changes to the documentation
- [na] I have added tests that prove my fix is effective or that my feature works
- [na] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

File: src/server/api/lib/nexmo.js
Comment (ibrand) [inline src/server/api/lib/nexmo.js:52]: The majority of this number isn't randomized anymore, what is `nexmo.js` used for? Will that be a problem at all?

**Review by ibrand — COMMENTED** [review]
Want to make sure we're not gonna run into problems with this lack of randomization (I'm guessing it's fine, but want to confirm before approval)

**Comment by schuyler1d** [comment]
Basically `rentNewCell` is completely dead code at the moment.  In both cases, the randomization is only done if an actual twilio account doesn't exist -- so that would be during testing.
So:
* dead code
* only used in testing
* if it were used in testing, and needed to get a new number, the result on the rest of the app is non-existent because we no longer send/pin by a cell number anyway

**Review by ibrand — APPROVED** [review]

**Pull request merged**
